modules/landing-pages/module.php

```diff
@@ -0,0 +1,342 @@
+<?php
+namespace Elementor\Modules\LandingPages;
+
+use Elementor\Core\Base\Document;
+use Elementor\Core\Base\Module as BaseModule;
+use Elementor\Core\Documents_Manager;
+use Elementor\Core\Experiments\Manager as Experiments_Manager;
+use Elementor\Modules\LandingPages\Documents\Landing_Page;
+use Elementor\Plugin;
+use Elementor\TemplateLibrary\Source_Local;
+use Elementor\User;
+use Elementor\Utils;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+class Module extends BaseModule {
+
+	const DOCUMENT_TYPE = 'landing-page';
+	const ADMIN_PAGE_SLUG = 'edit.php?post_type=page&elementor_library_type=landing-page';
+
+	private $posts;
+	private $trashed_posts;
+
+	public function get_name() {
+		return 'landing-pages';
+	}
+
+	public static function get_experimental_data() {
+		return [
+			'name' => 'landing-pages',
+			'title' => __( 'Landing Pages', 'elementor' ),
+			'description' => __( 'Adds a new Elementor content type that allows creating beautiful landing pages instantly in a streamlined workflow.', 'elementor' ),
+			'default' => Experiments_Manager::STATE_ACTIVE,
+		];
+	}
+
+	private function get_trashed_landing_page_posts() {
+		if ( $this->trashed_posts ) {
+			return $this->trashed_posts;
+		}
+
+		$this->trashed_posts = new \WP_Query( [
```

Comment:
If it's only an indicator that there are Landing Pages - please add `posts_per_page=1` for better performance.

Reply:
Good call, done.